sdk/src/main/java/co/omise/android/models/FlowType.kt

```diff
@@ -1,26 +1,42 @@
 package co.omise.android.models
 
+import android.annotation.SuppressLint
+import android.os.Parcel
 import com.fasterxml.jackson.annotation.JsonCreator
+import kotlinx.android.parcel.Parceler
 
 /**
  * Represents Source Flow object.
  *
  * @see [Sources API](https://www.omise.co/sources-api)
  */
 sealed class FlowType(
-        val name: String
+        val name: String?
 ) {
 
     object Redirect : FlowType("redirect")
     object Offline : FlowType("offline")
+    object Default : FlowType(null)
```

Comment:
`Default` looks misleading to me. I think we should choose a difference name for this?

Reply:
But I'm literally using it as Default for Jackson, what could we replace it with? `Unknown`? `Invalid`?

Reply:
Could it be `Unknown(value)` instead of null?

Reply:
No unfortunately, not with sealed classes. In any event, when Unknown is used in our code, there is no value to pass in anyway. I'll change the name to `Unknown` 👌